<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BDD</class>
 <widget class="QWidget" name="BDD">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BDD</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_NumDep">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_NomDerp">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>60</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ObtenirNom">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>210</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Obtenir Nom</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Quitter">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>240</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_NomDep">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>30</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom DÃ©partement</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_NumDep">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Numero Departement</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_Departement">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>79</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_Ville">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>160</y>
     <width>79</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_Region">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>79</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_Quitter</sender>
   <signal>clicked()</signal>
   <receiver>BDD</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>263</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
